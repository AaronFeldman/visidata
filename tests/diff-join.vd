sheet	col	row	longname	input	keystrokes	comment
			open-file	tests/data1.tsv	o	open input in VisiData
			open-file	tests/data2.tsv	o	open input in VisiData
data2	Key		key-col		!	pin current column on the left as a key column
data2			sheets		S	open Sheets Sheet
sheets		キdata1	open-row		^J	jump to sheet referenced in current row
data1	Key		key-col		!	pin current column on the left as a key column
data1			sheets		S	open Sheets Sheet
sheets		キdata1	select-row		s	select current row
sheets		キdata2	select-row		s	select current row
sheets			join-sheets	diff	&	merge the selected sheets with visible columns from all, keeping rows according to jointype
data1+data2	A		key-col		!	pin current column on the left as a key column
data1+data2			sort-keys-asc		g[	sort ascending by all key columns
data1+data2	A		key-col		!	unpin current column as a key column
