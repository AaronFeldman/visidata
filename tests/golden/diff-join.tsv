Key	data2_C	data2_D	data1_A	data1_B
1			a1	b1
3	c2	d2		
